src/client/websockets.js

```diff
@@ -397,6 +397,10 @@ WebSocketManager.prototype.onMessageReceived = function (message, content, role)
 
     content = content || [];
     if (message !== '') {
+        // if the message is for requestId
+        stage.threads.processes.forEach(function (p) {
+            if (p.requestId === message) p.reply = msg;
```

Comment:
y we need to change the message type to sth different than the message received. 
What do you think about a general message type of '__reply__' ?